### PR TITLE
EN6-182 Spaces on descriptions are not removed anymore

### PR DESCRIPTION
**Pull request by BrenoQVDS**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/content/widget/RowContentListHelper.java [hunk 1]
```diff
@@ -29,7 +29,11 @@
 		if (StringUtils.isBlank(contentsParameter)) {
 			return contents;
 		}
-		String parse1 = (contentsParameter.startsWith("[")) ? contentsParameter.substring(1) : contentsParameter;
+		String parse0 = contentsParameter
+				.replaceAll(", contentId=", ",contentId=")
+				.replaceAll(", modelId=", ",modelId=")
+				.replaceAll("\\}, \\{", "\\},\\{");
+		String parse1 = (parse0.startsWith("[")) ? parse0.substring(1) : parse0;
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/content/widget/RowContentListHelper.java]: Not the correct place to fix this. Please check `WidgetConfigPropertiesDeserializer`

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

**Pull request merged**